tdrs-frontend/src/components/EditProfile/EditProfile.jsx

```diff
@@ -1,65 +1,222 @@
     <div className="grid-container">
       <h1 className="request-access-header font-serif-2xl">Request Access</h1>
       <p className="request-access-secondary">
         We need to collect some information before an OFA Admin can grant you
         access
       </p>
-      <form className="usa-form">
-        <label className="usa-label" htmlFor="first-name">
-          First name
-          <input
-            className="usa-input"
-            id="firstName"
-            name="firstName"
-            type="text"
-            required
-            aria-required="true"
-          />
-        </label>
-
-        <label className="usa-label" htmlFor="lastName">
-          Last name
-          <input
-            className="usa-input"
-            id="lastName"
-            name="lastName"
-            type="text"
-            required
-            aria-required="true"
-          />
-        </label>
-        <ComboBox>
-          {stts.map((stt) => (
-            <option
-              className="sttOption"
-              key={stt.id}
-              value={stt.name.toLowerCase()}
-            >
-              {stt.name}
-            </option>
-          ))}
-        </ComboBox>
-        <Button
-          type="submit"
-          disabled
-          size="big"
-          className="request-access-button"
+      <form className="usa-form" onSubmit={handleSubmit}>
+        <div
+          className={`usa-form-group ${
+            errors.firstName ? 'usa-form-group--error' : ''
+          }`}
+        >
+          <label
+            className={`usa-label ${
+              errors.firstName ? 'usa-label--error' : ''
+            }`}
+            htmlFor="firstName"
+          >
+            First name
+            {errors.firstName && (
+              <span
+                className="usa-error-message"
+                id="input-error-message"
+                role="alert"
+              >
+                {errors.firstName}
+              </span>
+            )}
+            <input
+              className={`usa-input ${
+                errors.firstName ? 'usa-input--error' : ''
+              }`}
+              id="firstName"
+              name="firstName"
+              type="text"
+              aria-required="true"
+              value={profileInfo.firstName}
+              onChange={({ target: { value } }) => {
+                setProfileInfo({ ...profileInfo, firstName: value })
+              }}
+              onBlur={handleBlur}
+            />
+          </label>
+        </div>
+        <div
+          className={`usa-form-group ${
+            errors.lastName ? 'usa-form-group--error' : ''
+          }`}
+        >
+          <label
+            className={`usa-label ${errors.lastName ? 'usa-label--error' : ''}`}
+            htmlFor="lastName"
+          >
+            Last name
+            {errors.lastName && (
+              <span
+                className="usa-error-message"
+                id="input-error-message"
```

Comment:
@hilvitzs We'll want different ids for the different error messages.

Reply:
Thanks for pointing this out. I pushed the fix for both this and the other spot in combo box where this was happening.

Reply:
Excellent! Is the fix deployed to dev?

Reply:
Yes it is

Reply:
Thanks!